src/Http/Controllers/Auth/LoginController.php

```diff
@@ -79,6 +81,8 @@ public function showLoginForm()
         if (strlen(env('EVE_CLIENT_SECRET')) < 5 || strlen(env('EVE_CLIENT_ID')) < 5)
             session()->flash('warning', trans('web::seat.sso_config_warning'));
 
-        return view('web::auth.login');
+        $allow_registration = Seat::get('registration') === 'no' ? false : true;
```

Comment:
It can be shortened by `$allow_registration = setting('registration', true) !== 'no'`
Furthermore, use helper instead making direct call to this class.

https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L267